src/lib.rs

```diff
@@ -5,6 +5,11 @@
 //!
 #![deny(missing_debug_implementations)]
 
+#[cfg(not(any(target_os = "macos", target_os = "redox", windows)))]
```

Comment:
I didn't understand why it failed at first, but then I checked again what section I had added `dlib` to in the `Cargo.toml`. This should fix it.